libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +244,108 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+// send camera field of view status
+void AP_Camera_Backend::send_camera_fov_status(mavlink_channel_t chan)
+{
+    // POI has been requested
+    const AP_AHRS &ahrs = AP::ahrs();
+    Location l;
+    if (!ahrs.get_location(l)) {
+        gcs().send_text(MAV_SEVERITY_ERROR, "POI: vehicle pos unavailable");
+        return;
+    }
+
+    // change vehicle location to ASML
+    l.change_alt_frame(Location::AltFrame::ABSOLUTE);
+
+    // retrieve gimbal attitude
+    auto *mount = AP_Mount::get_singleton();
+    Vector3f att;
+    mount->get_attitude_euler(get_mount_instance(),att[0], att[1], att[2]);
```

Comment:
I think get_attitude_euler() returns a boolean so we should probably check the return.